third_party/move/move-compiler-v2/src/pipeline/ability_processor.rs

```diff
@@ -246,6 +246,22 @@ struct Transformer<'a> {
 
 impl<'a> Transformer<'a> {
     fn run(&mut self, code: Vec<Bytecode>) {
+        // Check and insert drop for parameters before the first instruction if it is a return (meaning the function body is empty)
```

Comment:
What is the difference between a function which has some instructions and one which is only one return? The body is not empty in either case.

Reply:
Sorry, the comment here is confusing. I think the case of only one return should be equivalent with the empty body where drops of parameters needs to be done before the return. If there are multiple instructions, drops of unused parameters can be handled by the existing code.